vsutil/__init__.py

```diff
@@ -279,13 +288,11 @@ def depth(clip: vs.VideoNode,
         return clip
 
     should_dither = _should_dither(curr_depth, bitdepth, range_in, range, clip.format.sample_type, sample_type)
-
     dither_type = fallback(dither_type, Dither.ERROR_DIFFUSION if should_dither else Dither.NONE)
 
-    return clip.resize.Point(format=clip.format.replace(bits_per_sample=bitdepth, sample_type=sample_type),
-                             range=range,
-                             range_in=range_in,
-                             dither_type=dither_type.value)
+    new_format = clip.format.replace(bits_per_sample=bitdepth, sample_type=sample_type).id
```

Comment:
You call `.id` on the new format here which didn’t happen before. Are both correct, and if so, why change it now?

Reply:
Because the typestubs do not include `Format` as a valid type to be passed to a vapoursynth filter.
It is supported though.
Mypy does however does not like it.

Reply:
I see. Thank you.